libindy/src/domain/ledger/ledgers_freeze.rs

```diff
@@ -0,0 +1,31 @@
+use super::constants::{LEDGERS_FREEZE, GET_FROZEN_LEDGERS};
+
+#[derive(Serialize, PartialEq, Debug)]
+pub struct LedgersFreezeOperation {
+    #[serde(rename = "type")]
+    pub _type: String,
+    pub ledgers_ids: Vec<u64>,
+}
+
+impl LedgersFreezeOperation {
+    pub fn new(ledgers_ids: Vec<u64>) -> LedgersFreezeOperation {
+        LedgersFreezeOperation {
+            ledgers_ids,
+            _type: LEDGERS_FREEZE.to_string()
```

Comment:
You changed the order of fields. I guess it will be better to return the order of values here.